docs/source/algorithms/MaskDetectors-v1.rst

```diff
@@ -6,35 +6,99 @@
 
 .. properties::
 
+Introduction
+------------
+
+To understand the algorithms options, user should clearly understand the difference between *WorkspaceIndex* 
+-- the numbers, specified in *WorkspaceIndexList* and *StartWorkspacIndex*, *EndWorkspaceIndex* properties,
+the *Spectra ID* -- values of the **SpectraList** property and *Detector ID* -- the numbers to provide for 
```

Comment:
There is various terminology within Mantid and in a wild (among IS), may be worth mentioning both terms.  as DetectorID is mentioned spectra ID refereed first here